src/test/java/bio/terra/pdao/bigquery/BigQueryPdaoTest.java

```diff
@@ -93,26 +105,98 @@ private String datasetName() {
         return "pdaotest" + StringUtils.remove(UUID.randomUUID().toString(), '-');
     }
 
+    private void AssertThatDatasetAndTablesShouldExist(Boolean shouldExist) {
```

Comment:
For consistency can you make this method name lowercase and switch from `Boolean` -> `boolean`?